Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.Shared/Controls/IControlWithValidationDispatcher.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.Development.WebTesting.TestSite.Shared.Controls
+{
+  public interface IControlWithValidationDispatcher
```

Comment:
`IDataEditControlWithValidationDispatcher` 
You could actually extend `IDataEditControl`, too. might make things simpler.

alternative: use `IUserControlWithValidationDispatcher`.

Personally, I prefer option one with extending IDataEditControl. It feels more like how we usually do things.